ui/app/src/modules/Content/Dependencies/DependenciesDialogUI.tsx

```diff
@@ -0,0 +1,473 @@
+                  </ListItemAvatar>
+                }
+                <ListItemText
+                  className={classes.listItemContent}
+                  primary={dependency.internalName}
+                  secondary={(!state.compactView) ? dependency.uri : null}
+                />
+
+                <ListItemSecondaryAction>
+                  <PopupState variant="popover">
+                    {popupState => (
+                      <React.Fragment>
+                        <IconButton
+                          edge="end"
+                          aria-label="delete"
+                          {...bindTrigger(popupState)}
+                        >
+                          <MoreVertIcon/>
+                        </IconButton>
+                        <Menu {...bindMenu(popupState)}>
+                          {
+                            isEditableItem(dependency.uri) &&
+                            <MenuItem onClick={() => {
+                              handleEditorDisplay(dependency);
+                              popupState.close();
+                            }}>
+                              <FormattedMessage
+                                id="dependenciesDialog.edit"
+                                defaultMessage="Edit"
+                              />
+                            </MenuItem>
+                          }
+                          <MenuItem onClick={() => {
+                            setState({ item: dependency });
+                            popupState.close();
+                          }}>
+                            <FormattedMessage
+                              id="dependenciesDialog.dependencies"
+                              defaultMessage="Dependencies"
+                            />
+                          </MenuItem>
+                          <MenuItem
+                            onClick={popupState.close}>   {/* TODO: pending, waiting for new history dialog */}
+                            <FormattedMessage
+                              id="dependenciesDialog.history"
+                              defaultMessage="History"
+                            />
+                          </MenuItem>
+                        </Menu>
+                      </React.Fragment>
+                    )}
+                  </PopupState>
+                </ListItemSecondaryAction>
+              </ListItem>
+            )
+          })
+      }
+    </List>
+  )
+}
+
+interface DependenciesDialogUIProps {
+  resource: Resource<Item[]>
+  state: any;
+  setState: Function;
+  open: boolean;
+  handleClose: any;
+  isEditableItem: Function;
+  assetsTypes: any;
+  editDialogConfig: any;
+  setEditDialogConfig: Function;
+  handleEditorDisplay: Function;
+}
+
+function DependenciesDialogUI(props: DependenciesDialogUIProps) {
+  const {
+    resource,
+    state,
+    setState,
+    open,
+    handleClose,
+    isEditableItem,
+    assetsTypes,
```

Comment:
This, being a static list could simply be an export. Let's actually colocate these two dialogs in the same file.